public/style.css

```
@@ -288,6 +300,12 @@ img {
     .field {
         margin: 20px 10px;
     }
+    .pull-left {
+          min-width: 50%;
+          display: flex;
+          justify-content: flex-end;
+          margin-left: 1.5em;
```

```suggestion
    .pull-left {
          margin-left: 1.5em;
```

I think this is mostly redundant. All you need here is `margin-left` here.

---

Done